Samples/IntegratedDependencies.Unity/Assets/NuGet.config

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <!--
+      Manually uncomment to reference the special feed of MSBuildForUnity NuGet package. This requirement will go away soon.
```

Comment:
How will this go away?

Reply:
Unfortunately, I'm being a parrot and simply repeating what's documented in the readme...

https://github.com/microsoft/MSBuildForUnity/blob/master/Samples/SimpleNuGetDependency.Unity/README.md

> Temporary: A NuGet.config that contains the special feed of MSBuildForUnity NuGet package. This requirement will go away soon.

I'm not tracking any work to resolve the above.

Reply:
Maybe the idea was to publish to npm.com, can’t remember 🤷‍♂️